x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/job/process/autodetect/output/AutodetectResultProcessor.java

```diff
@@ -310,6 +325,21 @@ void processResult(AutodetectResult result) {
         }
     }
 
+    private Annotation createModelSnapshotAnnotation(ModelSnapshot modelSnapshot) {
+        assert modelSnapshot != null;
+        Date currentTime = new Date(clock.millis());
+        return new Annotation(
+            Messages.JOB_AUDIT_SNAPSHOT_STORED,
```

Comment:
Can we also include the model snapshot ID, so that if somebody wants to revert to the snapshot they can get the piece of information they need in order to do this from the annotation?